cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -217,8 +217,8 @@ type Telemetry struct {
 }
 
 type Logs struct {
-	Level       zapcore.Level `mapstructure:"level"`
 	OutputPaths []string      `mapstructure:"output_paths"`
+	Level       zapcore.Level `mapstructure:"level"`
```

Comment:
Is this done by a tool? Looks like it simply orders the field by the size, but it doesn't matter in this case. It should be 32 bits in both cases. Curious if it ca be configurable to avoid changing fields if it's unnecessary?

I'm a bit concerned because it's technically a public API-breaking change for someone initializing it as Logs{level, paths} even if it's an anti-pattern.

Reply:
this is done by `fieldalignment -fix ./...`.